api/src/test/java/com/epam/pipeline/acl/issue/IssueApiServiceTest.java

```diff
@@ -326,4 +331,12 @@ public void shouldDenyDeleteCommentWhenPermissionIsNotGranted() {
 
         assertThrows(AccessDeniedException.class, () -> issueApiService.deleteComment(ID, ID));
     }
+
+    @Test
+    @WithMockUser
+    public void shouldLoadMyPagedResultListIssue() {
+        final PagedResult<List<Issue>> pagedResult = IssueCreatorUtils.getPagedListIssue();
+        doReturn(pagedResult).when(mockIssueManager).loadMy(ID, TEST_INT);
+        assertThat(issueApiService.loadMy(ID, TEST_INT)).isEqualTo(pagedResult);
```

Comment:
Please extract the `pagedResult` field and add line break between mocking and asserting.